src/clj_meowallet/http.clj

```diff
@@ -30,37 +32,68 @@
     (json/generate-string body)
     body))
 
+(defn get-status-from-error
+  [response]
+  (-> (.getMessage response)
+      (str/split #" ")
```

Comment:
This seems weak. Response is an exception? What if the message is not what you are expecting?